docs/aws-sdk-for-dotnet-with-minio.md

```diff
@@ -24,42 +24,52 @@ The example prints all buckets in the Minio server and lists all objects of the
 ```csharp
 using Amazon.S3;
 using System;
+using System.Threading.Tasks;
+using Amazon;
 
 class Program
 {
     private const string accessKey = "PLACE YOUR ACCESS KEY HERE";
     private const string secretKey = "PLACE YOUR SECRET KEY HERE"; // do not store secret key hardcoded in your production source code!
-    
+
     static void Main(string[] args)
     {
-        var amazonS3Client = new AmazonS3Client(accessKey, secretKey, new AmazonS3Config { ServiceURL = "http://localhost:9000", ForcePathStyle = true }); // replace http://localhost:9000 with URL of your minio server
+        Task.Run(MainAsync).GetAwaiter().GetResult();
+    }
+
+    private static async Task MainAsync()
+    {
+        var config = new AmazonS3Config
+        {
+            RegionEndpoint = RegionEndpoint.USEast1, // MUST set this before setting ServiceURL

```

Comment:
From https://github.com/minio/minio/issues/4653#issuecomment-314209518 it appears that this is required otherwise will get signature mismatch exceptions. Note: there appears to be a big where it only seems to work with region `USEast1` AND with setting the `MINIO_REGION` environment variable.